projects/dashboards/ecometrics/src/app/app.component.html

```diff
@@ -0,0 +1,5 @@
+<h1 class="mat-h1">
+    <img src="assets/geoengine.svg" class="logo" />
+    <div>ECOMETRICS</div>
+</h1>
+<app-dashboard></app-dashboard>
```

Comment:
![grafik](https://github.com/user-attachments/assets/fe336429-084a-4348-942a-ed31ae26cf0a)

For normal browser sizes, the selector overflows

Reply:
the indicicator could still overflow so I moved the descritption into the select

9bfc35ffb6a5bafc995da0c6b006f1ce554e3cc5
![image](https://github.com/user-attachments/assets/bef58ec9-2e1b-49d9-af9b-79bd2b7815fb)

Reply:
We could merge it for now.

But I guess we should use a CSS grid with flexible height, instead.

Reply:
yes I think that's the right move. I added an item in the checklist